scss/civicrm/_variables.scss

```diff
@@ -15,6 +15,7 @@ $crm-form-layout-shadow: 0 3px 18px 0 rgba(48, 40, 40, 0.25);
 $crm-input-shadow: 0 0 3px 0 rgba(0, 0, 0, 0.2) inset;
 $crm-search-input-shadow: 0 1px 1px rgba(0, 0, 0, 0.075) inset;
 $crm-tab-shadow: 0 9px 9px 0 rgba(48, 40, 40, 0.25);
+$crm-panel-element-shadow: 0 3px 18px 0px rgba(0, 0, 0, .1);
```

Comment:
why do we have so many different shadows?

Reply:
The $crm-panel-element-shadow I am using because we have several elements right one after the other and each one needs to have a background color and a shadow due to the lack of a common container element for all these elements. This shadow visually looks like the design, so that's why I have added it.

Reply:
Here we can use variable $box-shadow if we need to add shadow, or this is different?